### PR TITLE
Impl constants module

### LINKED ISSUE #46: add constant module

### PR DESCRIPTION
**Pull request by terasakisatoshi**
resolve #46 

Example

```julia
julia> using SciPy
julia> pyconstants

PyObject <module 'scipy.constants' from '~/.asdf/installs/python/3.8.10/lib/python3.8/site-packages/scipy/constants/__init__.py'>

julia> typeof(constants)
Module

julia> pyconstants.golden
1.618033988749895

julia> constants.golden
1.618033988749895
```

### CODE REVIEW

File: src/scipy_api_list.py [hunk 1]
```diff
@@ -56,11 +56,15 @@ def generate_scipy_apis(pymod) -> dict:
...
     for attr in attributes:
+        is_const = True
         for (key, method) in types:
             condition = eval(f"inspect.{method}(pymod.{attr})")
             if condition:
                 d[key].append(attr)
+                is_const = False
                 break
+        if is_const:
+            d["constant"].append(attr)
```
Comment (AtsushiSakai) [inline src/scipy_api_list.py]: [Question] Why this handling is needed? the constant module is different from other modules?

**Review by AtsushiSakai — COMMENTED** [review]
Is it possible to add a document section for the `constant` module?
image

**Comment by terasakisatoshi** [comment]
> Is it possible to add a document section for the `constant` module?

Did you mean `constants`? We already have the pair object `"constants" => "example/constants.md"`

https://github.com/AtsushiSakai/SciPy.jl/blob/3caf6d25e58c371e8db1eced88e085ede4efd16e/docs/make.jl#L21

**Comment by AtsushiSakai** [comment]
Sorry, I totally forget it. Never mind.

File: src/scipy_api_list.py (hunk 1 above)
Comment (terasakisatoshi) [reply src/scipy_api_list.py]: Let's say we want to generate API to get mathematical/physical constants e.g. `scipy.constants.golden_ratio` and `scipy.constants.gram`. They are defined in `scipy/scipy/constants/_constants.py`:

https://github.com/scipy/scipy/blob/74946d2b20d03af9c5b6caa0d34b8c76a52333c8/scipy/constants/_constants.py#L68-L76

https://github.com/scipy/scipy/blob/74946d2b20d03af9c5b6caa0d34b8c76a52333c8/scipy/constants/_constants.py#L122

We note that they are defined in the form of `<variable_name> = <value>`, which is not treated as a `function`, `module` nor `class`, which means the Python module `inspection` can't capture them.

Therefore to collect value of constants, I have to update `generate_scipy_apis` at least.

**Comment by terasakisatoshi** [comment]
As [we discussed here](https://twitter.com/MathSorcerer/status/1581597526368866305?s=20&t=gpRvNoCC4tIPSXpXJIA9sg), we found we did not have to `src/scipy_api_list.py` at all. 

For `f` in `pyconstants.__all__`, we only have to check whether the variable `a` below can be converted in a Julia's native type (i.e. not `PyObject`):

```julia
a = pybuiltin("getattr")(pyconstants, f)
```

This trick should work for `constants` module 😄.

**Comment by AtsushiSakai** [comment]
Rerun CIs

**Review by AtsushiSakai — APPROVED** [review]
LGTM. Let's merge it.

**Comment by terasakisatoshi** [comment]
🚀

**Comment by AtsushiSakai** [comment]
@terasakisatoshi I think we are ready to release v0.1.2. Do you have time to release it?

**Pull request merged**